contracts/exchange/api/SubAccountContract.sol

```diff
@@ -56,39 +56,6 @@ contract SubAccountContract is BaseContract, ConfigContract, FundingAndSettlemen
     acc.subAccounts.push(subAccountID);
   }
 
-  /// @notice Change the margin type of a subaccount
```

Comment:
A bit hesitance to remove this since we already got this reviewed.

Reply:
reverted the changes in this file.